config/profile.example.yml

```
@@ -132,3 +132,21 @@ cover_letter:
+#   economy  → claude-3-5-haiku-latest   (~$0.01–0.03 / evaluation)
+#   standard → claude-sonnet-4-6          (~$0.05–0.15 / evaluation)
+#   premium  → claude-opus-4              (~$0.20–0.50 / evaluation)
+#
+# Standard and premium also apply a cheap Haiku pre-screen gate in batch mode:
+# a Haiku worker scores each offer first; only offers scoring >= 3.0 are
+# forwarded to the main model, saving tokens on obvious mismatches.
+#
+# Omitting this key (or leaving it commented out) is equivalent to "standard".
+#
+spend_tier: standard  # economy | standard | premium
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)